Sources/Fuzzilli/Lifting/JavaScriptRuntimeAssistedMutatorLifting.swift

```diff
@@ -511,6 +512,7 @@ struct JavaScriptRuntimeAssistedMutatorLifting {
           [OP_LOGICAL_AND]: (inputs) => inputs[0] && inputs[1],
           [OP_LOGICAL_OR]: (inputs) => inputs[0] || inputs[1],
           [OP_LOGICAL_NOT]: (inputs) => !inputs[0],
+          [OP_LOGICAL_NullCoalesce]: (inputs) => inputs[0] ?? inputs[1],
```

Comment:
This is good, but to be useful, you also need to create these operations. I think mainly this is useful for the ExplorationMutator which creates new instructions depending on the runtime type of variables. Concretely, what I would suggest is replacing this here: https://github.com/googleprojectzero/fuzzilli/blob/5798638831dbc3347a0aa557f14ef63ebed704b3/Sources/Fuzzilli/Lifting/JavaScriptExploreLifting.swift#L250 and here: https://github.com/googleprojectzero/fuzzilli/blob/5798638831dbc3347a0aa557f14ef63ebed704b3/Sources/Fuzzilli/Lifting/JavaScriptExploreLifting.swift#L425 to call a new `exploreNullish` function which then generates this operation. Something like:
```
function exploreNullish(v) {
    // Best thing we can do with nullish values is a NullCoalescing (??) operation.
    return new Action(OP_NULL_COALESCE, [exploredValueInput, Inputs.randomArgument()])
}
```

Then this mutator should be able to generate these new operations :)

Reply:
Thank you for your valuable advice. I did miss many details. I have modified and then pushed it.